libs/application/template-api-modules/src/lib/modules/shared/services/attachment-s3.service.ts

```diff
@@ -43,45 +51,94 @@ export class AttachmentS3Service {
       name: string
     }>,
     answerKey: string,
-    application: Application,
+    application: ApplicationWithAttachments,
   ): Promise<AttachmentData[]> {
     return await Promise.all(
       answers.map(async ({ key, name }) => {
-        const url = (
-          application.attachments as {
-            [key: string]: string
-          }
-        )[key]
-
+        const url = this.getFilenameFromApplication(application, key)
```

Comment:
Tiny bit confusing that `getFilename...` returns a url

Reply:
Satt, breytti variable name í eh skiljanlegra vonandi, copy-paste function frá öðrum class 😅